### DISCUSSION TITLE
vc-drawings相关问题

### DISCUSSION THREAD

**Discussion by xiaobaicxk** [zouyaoji/vue-cesium#449] [Q&A] [answered]
请问大佬一下，带图片的标注点位在地图元素加载完成后，是否还能对标注点动态的隐藏或者显示根据对应图标点的名称或者什么参数之类的，这个目前框架中有办法实现这种效果吗。

类似于下图的点击事件后控制对应的图标点隐藏显示
![image](https://github.com/zouyaoji/vue-cesium/assets/103714416/e20e8406-20fb-40f0-afb6-5d417d9a4008)

通过尝试对drawingsRef.value.getDrawingActionInstances()[0].cmpRef.value.renderDatas中属性有show的修改
![image](https://github.com/zouyaoji/vue-cesium/assets/103714416/dbb41924-93a7-44e7-b89a-50dab7c5e744)
无法达到我预期的通过点击事件修改后隐藏对应类型或者名称的图标点，然后再通过某个点击事件显示对应类型或者名称的图标点。

**Comment by zouyaoji** [comment, marked as answer]
drawingsRef.value.getDrawingActionInstances()[0].cmpRef.value.renderDatas应该是能的哈，你要尝试把ui上的checkbox的model绑定上去。

这种为啥不用vc-datasource-cutom组件呢？

比如参考https://zouyaoji.top/vue-cesium-demo 这里面有一定的封装，结合绘制组件的话，绘制组件只是绘制，绘制完之后还是存到 vc-datasource-cutom的entities中去。

**Reply by xiaobaicxk** [reply]
感谢大佬 我试试~
